### PR TITLE
Email settings documentation

### LINKED ISSUE #3141: flash[:warning] appearing on subsequent pages
Hi, @Gauravano - this isn't exclusively related to drafts, but it did come up there. I've noticed flash messages get "delayed" and shown on the wrong page, say, on subsequent page loads on unrelated pages. Can you try to reproduce this? It did happen when viewing a draft page. I think it may happen for other interactions as well. 

I'd love some help trying to track this down and solving it! Thanks!

### PR DESCRIPTION
**Pull request by gauravano**
Fixes #3141 

Make sure these boxes are checked before your pull request (PR) is ready to be reviewed and merged. Thanks!

* [x] tests pass -- look for a green checkbox ✔️ a few minutes after opening your PR -- or run tests locally with `rake test`
* [x] code is in uniquely-named feature branch and has no merge conflicts
* [x] PR is descriptively titled
* [x] PR body includes `fixes #0000`-style reference to original issue #
* [x] ask `@publiclab/reviewers` for help, in a comment below

> We're happy to help you get this ready -- don't be afraid to ask for help, and **don't be discouraged** if your tests fail at first!

If tests do fail, click on the red `X` to learn why by reading the logs.

Please be sure you've reviewed our contribution guidelines at https://publiclab.org/contributing-to-public-lab-software

We have a loose schedule of reviewing and pulling in changes every Tuesday and Friday, and publishing changes on Fridays. 

Thanks!

### CODE REVIEW

**Comment by Gauravano** [comment]
Hi @jywarren could you please review the documentation, should I also add the location where I have used these conditions to control settings? Thanks!

File: doc/EMAIL.md [hunk 1]
```diff
@@ -8,4 +8,38 @@ After configurations are done we are good to start the `mailman` server. For sta
...
+
+Users can modify **Email Settings** by visiting [https://publiclab.org/settings](https://publiclab.org/settings). Settings for each user are saved by using UserTags on their profiles. 
+For each setting, there is an associated tag with it. Below is the table showing associated tags with setting. 
+
+We consider absence of tag as true and presence of tag as false for setting, so whenever a user turn-off a setting, corressponding user-tag is generated and turning setting on again will delete the tag.  
+So, while notifying a user UserTag is checked in different files and user is notified depending on the presence or absence of the tag.
+
+| Email Settings  | User Tag |
+| ------------- | ------------- |
+| Notification by email for comments on your posts  | notify-comment-direct:false |
```
Comment (jywarren) [inline doc/EMAIL.md]: Looks great, maybe we can have a column like `Default` showing the default setting, and the `User Tag` could say `User tag to override default` ? What do you think?

Comment (jywarren) [reply doc/EMAIL.md]: and, i like the idea of linking to the lines of code!

Comment (Gauravano) [reply doc/EMAIL.md]: ok, I will make the changes. Thanks!

**Comment by Gauravano** [comment]
@jywarren plz review this PR. Thanks!

**Comment by Gauravano** [comment]
@Gauravano

**Comment by Gauravano** [comment]
@gauravano

**Comment by jywarren** [comment]
Awesome. Thanks!!!

Also, nice new picture! Maybe time to update mine! 📸

**Comment by gauravano** [comment]
Thanks @jywarren 😬 ! Definitely, go for it!

**Pull request merged**